src/main/python/vantiqsdk.py

```diff
@@ -1238,7 +1238,8 @@ async def start_subscriber_transport(self):
             return None
 
     async def subscribe(self, resource: str, resource_id: str, operation: Union[str, None],
-                        callback: Callable[[str, dict], Awaitable[None]], params: dict = None) -> VantiqResponse:
+                        callback: Callable[[str, dict], Awaitable[None]], params: Union[dict, None] = None,
+                        target_namespace: Union[str, None] = None) -> VantiqResponse:
```

Comment:
Add `target_namespace`

Reply:
is there a reason that we use `register_subscriber_on_close()` vs. calling the existing callback with a `close` reason?  I assume for BW compatibility (existing handlers get an unexpected `close` call).

Reply:
The existing handlers are also per-subscription.  This is for when the entire subscriber goes away (so it is much easier for the client to handle one callback rather than 20 which say the same thing).  Happy to switch if you disagree, but it seemed that they were targeting different things.